pulsar-broker/src/main/java/org/apache/pulsar/PulsarClusterMetadataSetup.java

```diff
@@ -338,6 +380,32 @@ static void createNamespaceIfAbsent(ZooKeeper configStoreZk, NamespaceName names
         }
     }
 
+    static void createNamespaceIfAbsent(MetadataStore configStore, NamespaceName namespaceName, String cluster)
```

Comment:
Can we delete this one? https://github.com/apache/pulsar/pull/10600/files#diff-03a1201aab4aa284723e575a7c9b18ad9542fcc02beb80d83cfb87d64adf54b5R353

Reply:
It's referenced in `PulsarInitialNamespaceSetup`, I think it's better to open a new PR for setup namespace with MetadataStore API.

Reply:
Yes, as I explained above.